group_members.go

```diff
@@ -258,14 +258,14 @@ func (s *GroupsService) ListBillableGroupMembers(gid interface{}, opt *ListBilla
 //
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/members.html#list-memberships-for-a-billable-member-of-a-group
-func (s *GroupsService) ListMembershipsForBillableGroupMember(gid interface{}, user int, options ...RequestOptionFunc) ([]*BillableUserMembership, *Response, error) {
+func (s *GroupsService) ListMembershipsForBillableGroupMember(gid interface{}, user int, opt *ListOptions, options ...RequestOptionFunc) ([]*BillableUserMembership, *Response, error) {
```

Comment:
Please check the code for examples, but you should not use the `ListOptions` struct directly. Instead you should define a `ListMembershipsForBillableGroupMemberOptions` struct which could be just `type ListMembershipsForBillableGroupMemberOptions = ListOptions`...

Reply:
Ah, ok - wasn't sure about that, as I found some other occurrences, where it's used directly (e.g. in `ListProjectStatusChecks`) 
I'll change it as suggested.

Reply:
As far as I can see there are 3 other location where it slipped in, but good to not let the number increase 😏